### PR TITLE
GEN-1233 - feat(TrialExtensionForm): added a sign disclaimer

### PR DESCRIPTION
**Pull request by guilhermespopolin**
## Describe your changes

- Added a sign disclaimer - as we already have for CheckoutPage and ManyPets offer page

Screenshot 2023-09-29 at 16 56 49

## Justify why they are needed

Requested by design

### CODE REVIEW

**Comment by guilhermespopolin** [comment]
Current dependencies on/for this PR:
* main
  * **PR #3299** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3299" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3299?utm_source=stack-comment).

File: apps/store/src/features/carDealership/TrialExtensionForm.tsx [hunk 1]
```diff
@@ -175,12 +180,25 @@ export const TrialExtensionForm = (props: Props) => {
...
-            </Button>
+            <Space y={1}>
+              <Button onClick={handleSignAndPay} loading={loading}>
+                <SpaceFlex space={0.5} align="center">
+                  <BankIdIcon />
+                  {props.requirePaymentConnection ? SIGN_AND_PAY_BUTTON : SIGN_BUTTON}
+                </SpaceFlex>
+              </Button>
+
+              <TextWithLink
```
Comment (robinandeer) [inline apps/store/src/features/carDealership/TrialExtensionForm.tsx:191]: I would consider adding `balance=true` 🙂

File: apps/store/src/features/carDealership/TrialExtensionForm.tsx [hunk 2]
```diff
@@ -35,6 +38,8 @@ type Props = {
 
 export const TrialExtensionForm = (props: Props) => {
   const [userWantsExtension, setUserWantsExtension] = useState(true)
+  const { t } = useTranslation('checkout')
```
Comment (robinandeer) [inline apps/store/src/features/carDealership/TrialExtensionForm.tsx:41]: yeah this is for sure becoming confusing hehe

but no need to change now

File: apps/store/src/features/carDealership/TrialExtensionForm.tsx [hunk 3]
```diff
@@ -1,14 +1,17 @@
 import { datadogRum } from '@datadog/browser-rum'
 import { useRouter } from 'next/router'
 import { useState, useMemo } from 'react'
+import { useTranslation } from 'react-i18next'
```
Comment (robinandeer) [inline apps/store/src/features/carDealership/TrialExtensionForm.tsx:4]: this should be "next-i18next"

**Review by robinandeer — APPROVED** [review]

**Comment by guilhermespopolin** [comment]
### Merge Activity

* **Oct 2, 4:44 AM**: [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3299) rebased this pull request as part of a merge.

**Pull request merged**